core/src/main/scala/org/broadinstitute/dsde/rawls/serviceFactory/BucketMigrationServiceFactory.scala

```diff
@@ -0,0 +1,24 @@
+package org.broadinstitute.dsde.rawls.serviceFactory
+
+import org.broadinstitute.dsde.rawls.bucketMigration.{BucketMigration, BucketMigrationService}
+import org.broadinstitute.dsde.rawls.config.RawlsConfigManager
+import org.broadinstitute.dsde.rawls.dataaccess.{GoogleServicesDAO, SamDAO, SlickDataSource}
+import org.broadinstitute.dsde.rawls.model.RawlsRequestContext
+import org.broadinstitute.dsde.rawls.model.WorkspaceCloudPlatform.{Azure, Gcp}
+import org.broadinstitute.dsde.rawls.serviceFactory.DisabledServiceFactory.newDisabledService
+
+import scala.concurrent.ExecutionContext
+
+object BucketMigrationServiceFactory {
+  def createBucketMigrationService(appConfigManager: RawlsConfigManager,
```

Comment:
Maybe this should be called `createBucketMigration` to match the name of the interface it returns?